src/main/MQ2ImGuiConsole.cpp

```diff
@@ -909,6 +914,7 @@ class ImGuiConsole
 	std::vector<std::string> m_history;
 	int m_historyPos = -1;    // -1: new line, 0..History.Size-1 browsing history.
 	bool m_scrollToBottom = true;
+	bool AlphaBool = false;
```

Comment:
as a member variable this should follow the convention and start with `m_`. It should also have a more accurate and descriptive name. I'm not sure what an "AlphaBool" does by reading it.